packages/web-js-sdk/src/enhancers/withLastLoggedInUser/types.ts

```diff
@@ -0,0 +1,11 @@
+export type LastLoggedInUserOptions = {
+  // If true, the last authenticated user will be stored in local storage
+  // and will be used in the next flow start request
+  // Default is true
+  storeLastAuthenticatedUser?: boolean;
+  // If true, the last authenticated user will be kept in local storage even after logout
+  // and will be used in the next flow start request
+  // This option is relevant only if storeLastAuthenticatedUser is true
+  // Default is false
+  keepLastAuthenticatedUserAfterLogout?: boolean;
```

Comment:
is this option make sense even if `storeLastAuthenticatedUser` is not in used?
![image](https://github.com/descope/descope-js/assets/10514677/f5523117-f2c7-4a9d-aad4-f8a6cdb4fd9f)

if it doens't make snese to use it without `storeLastAuthenticatedUser` maybe have it as conditional to `storeLastAuthenticatedUser: true`

not a must ofc

Reply:
it will just not delete stuff that are not there, so should be good